schemas/io-package.json

```diff
@@ -187,6 +187,16 @@
             }
           }
         },
+        "automaticUpgrade": {
```

Comment:
make it optional?

Reply:
it is - as it is not included in the `required` ones